app.py

```
@@ -0,0 +1,30 @@
+from docx_to_markdown import convert_docx_to_markdown
+
+app = Flask(__name__)
+UPLOAD_FOLDER = 'uploads'
+os.makedirs(UPLOAD_FOLDER, exist_ok=True)
+
+@app.route('/')
+def index():
+    return send_file('index.html')
+
+@app.route('/convert', methods=['POST'])
+def convert():
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)